### PR TITLE
Add Protected Species Landing Page

### PR DESCRIPTION
**Pull request by redrambles**
We have added a Protected Species landing page, which shows all species that have a protected status (CITES, CMS).

The status has a tooltip, we would need the actual help text here.

image

The CITES and CMS statuses can be added to Species in the Admin dashboard.

image

### CODE REVIEW

File: app/controllers/protected_species_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class ProtectedSpeciesController < ApplicationController
```
Comment (coffeejunk) [inline app/controllers/protected_species_controller.rb]:
```suggestion
class ProtectedSpeciesController < PreAuthController
```

**Review by coffeejunk — CHANGES_REQUESTED** [review]

**Comment by coffeejunk** [comment]
@creeas any thoughts? 🙂

File: Gemfile.lock [hunk 2]
```diff
@@ -448,8 +450,9 @@ DEPENDENCIES
   actionpack-action_caching (~> 1.2)
   activerecord-postgis-adapter (~> 8.0.0)
   administrate (~> 0.18.0)
-  after_commit_everywhere (~> 1.3)
-  aws-sdk-s3 (~> 1.117)
+  administrate-field-enum
+  after_commit_everywhere (~> 1.2)
+  aws-sdk-s3 (~> 1.114)
```
Comment (coffeejunk) [inline Gemfile.lock]:
```suggestion
  aws-sdk-s3 (~> 1.117)
```

File: app/javascript/styles.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+@import "~bulma/bulma";
+
+$fa-font-path: "@fortawesome/fontawesome-free/webfonts";
+@import "@fortawesome/fontawesome-free/scss/fontawesome";
```
Comment (coffeejunk) [inline app/javascript/styles.scss]: I dropped bulma & fontawesome from the pack, can you load this from via [CDN](https://cdnjs.com/libraries/bulma-tooltip)?

File: Gemfile.lock [hunk 4]
```diff
@@ -89,7 +89,9 @@ GEM
       kaminari (>= 1.0)
       sassc-rails (~> 2.1)
       selectize-rails (~> 0.6)
-    after_commit_everywhere (1.3.0)
+    administrate-field-enum (0.0.9)
+      administrate (~> 0.12)
+    after_commit_everywhere (1.2.2)
```
Comment (coffeejunk) [inline Gemfile.lock]:
```suggestion
    after_commit_everywhere (1.3.0)
```

File: package.json [hunk 5]
```diff
@@ -19,6 +19,8 @@
   },
   "dependencies": {
     "@babel/preset-react": "^7.18.6",
+    "@creativebulma/bulma-tooltip": "^1.2.0",
+    "@fortawesome/fontawesome-free": "^6.2.0",
```
Comment (coffeejunk) [inline package.json]:
```suggestion
```

load via CDN as mentioned above

File: yarn.lock [hunk 6]
```diff
@@ -1314,6 +1314,11 @@
   resolved "https://registry.yarnpkg.com/@bcoe/v8-coverage/-/v8-coverage-0.2.3.tgz#75a2e8b51cb758a7553d6804a5932d7aace75c39"
   integrity sha512-0hYQ8SB4Db5zvZB4axdMHGwEaQjkZzFjQiN9LVYvIFB2nSUHW9tYpxWriPrWDASIxiaXax83REcLxuSdnGPZtw==
 
+"@creativebulma/bulma-tooltip@^1.2.0":
+  version "1.2.0"
+  resolved "https://registry.yarnpkg.com/@creativebulma/bulma-tooltip/-/bulma-tooltip-1.2.0.tgz#84dcdd59d94c09c2975fadbec1d3d765ae29c471"
+  integrity sha512-ooImbeXEBxf77cttbzA7X5rC5aAWm9UsXIGViFOnsqB+6M944GkB28S5R4UWRqjFd2iW4zGEkEifAU+q43pt2w==
+
```
Comment (coffeejunk) [inline yarn.lock]:
```suggestion
```

**Comment by creeas** [comment]
> @creeas any thoughts? 🙂

Hey @coffeejunk and @redrambles this looks really great. I like the easy edit in the admin dashboard as well which will make it easier to edit in the future. I will generate some text for each appendix for CITES and CMS and get them to you.

**Comment by coffeejunk** [comment]
note to self & @redrambles: #828

**Review by coffeejunk — APPROVED** [review]

**Pull request merged**